### PR TITLE
Routing encryption

### PR DESCRIPTION
**Pull request by bfallonf**
Changes to the routing.adoc file as per edits and chatting to devs.

Pinging @pweil- for a tech review.

Thanks!

### CODE REVIEW

**Comment by bfallonf** [comment]
@pweil- 

I've made a few edits of my own (sentence level) and it should be largely the same. However, I'm concerned I've maybe confused the Overview section in the trying to understand it myself.

File: architecture/routing.adoc [hunk 1]
```diff
@@ -10,21 +10,18 @@
 toc::[]
 
 == Overview
-Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information
-directly to the router (HTTP, HTTPS, and TLS with SNI). Routers subscribe to configuration changes and automatically
-update themselves with new configuration, and routers may be containerized or virtual (converting those changes to API calls to a system like an F5).
+Routers provide external DNS mapping and load balancing to services over protocols that pass distinguishing information directly to the router (HTTP, HTTPS, and TLS with SNI). Routers subscribe to a configuration and automatically update themselves with any changes, and may be containerized or virtual (converting those changes to API calls to a system like an F5).
```
Comment (pweil-) [inline architecture/routing.adoc]: Implementation detail that may be a little confusing, not sure if we want to make the distinction or not....In the current implementation of the router it doesn't actually forward traffic to the service.  By that, I mean, it does not use the service ip/port when forwarding traffic.  It forwards traffic to the endpoints that back the service.  It only uses the service selector to watch the endpoints that back the service.  (It's ultimately the same thing, but bypasses the service load balancing logic and replaces it with its own. Maybe it isn't worth mentioning)

File: architecture/routing.adoc [hunk 2]
```diff
@@ -61,19 +60,19 @@ Example secure route (using edge termination)
...
+
 == Securing Routes
-Creating a secure route to your pods can be accomplished by specifying the TLS Termination of the route and, optionally,
-providing certificates to use. As of writing, OpenShift beta1 TLS termination relies on SNI for serving custom certificates (in the reference HAProxy implementation).
-Any non-SNI traffic received on port 443 will have TLS terminated with a generic certificate.  In the future, the ability
-to create custom frontends within the router will allow all traffic to serve custom certificates.
+Creating a secure route to your pods is accomplished by specifying the TLS Termination of the route and, optionally, providing certificates. At the time of writing, OpenShift Beta 1 TLS termination relies on SNI for serving custom certificates. Any non-SNI traffic received on port 443 has TLS termination with a generic certificate. In the future, the ability to create custom frontends within the router will allow all traffic to serve custom certificates.
 
-There are three types of TLS termination:
+By default, OpenShift has no pod encryption, but can be set to any of the following three types of TLS termination:
```
Comment (pweil-) [inline architecture/routing.adoc]: I would remove the reference to pod here, we're trying to keep from directly tying a route to a pod.  Perhaps, "by default, OpenShift routes are unsecure"

**Comment by bfallonf** [comment]
@pweil- I've made two edits after your suggestions here. 

With your first, I added in a sentence "With a service, the router uses the service selector to watch the endpoints that support the service, bypassing its logic and replacing it with the router’s own."

And the second is just as suggested. 

Please, let me know if there are any other suggestions for this pull request, otherwise I can send this off.

Thanks!

**Comment by pweil-** [comment]
LGTM

**Pull request merged**